### PR TITLE
feat: rate comic

### LINKED ISSUE #156: Rate comic

### PR DESCRIPTION
**Pull request by basmasking**
Fixes #156 

@MaskingTechnology/comify

### CODE REVIEW

File: src/domain/post/data/retrieve.ts [hunk 1]
```diff
@@ -1,7 +1,14 @@
 
+import database from '../../../integrations/database/module';
+import { RECORD_TYPE } from '../definitions/constants';
 import PostData from './PostData';
+import mapRecord from './mapRecord';
 
 export default async function retrieve(id: string): Promise<PostData>
 {
-    return new PostData(id, 'creatorId', 'comicId');
+    console.log('retrieve', id);
```
Comment (petermasking) [inline src/domain/post/data/retrieve.ts:9]:
```suggestion
```

File: test/domain/post/_fixtures/toggleRating.fixture.ts
Comment (petermasking) [inline test/domain/post/_fixtures/toggleRating.fixture.ts:1]: Is there a specific reason why the toggle rating function has it's own fixture? Ow can we merge it into the post fixture?

File: test/domain/post/toggleRating.spec.ts
Comment (petermasking) [inline test/domain/post/toggleRating.spec.ts:1]: The rollback of the saga isn't tested in this spec yet. Can you add one? For an example you can look at the `establish.spec` file in the relation module.

**Review by petermasking — CHANGES_REQUESTED** [review]

File: test/domain/post/_fixtures/toggleRating.fixture.ts
Comment (basmasking) [reply test/domain/post/_fixtures/toggleRating.fixture.ts:1]: As discussed, we leave this for now and discuss on Monday

**Review by petermasking — APPROVED** [review]

**Pull request merged**